app/src/main/res/values/strings.xml

```diff
@@ -468,6 +468,7 @@
   <string name="mark_stories_completed_activity_title">Mark Stories Completed</string>
   <string name="mark_topics_completed_activity_title">Mark Topics Completed</string>
   <string name="view_event_logs_activity_title">View Event Logs</string>
+  <string name="force_network_type_activity_title">Force Network Type</string>
```

Comment:
I wonder if this should be 'Mode' instead of 'Type' since that might be a bit clearer, but I'm not sure.

@rt4914 any thoughts?

Reply:
I don't have a strong opinion here; resolving to unblock the PR.